flashlib.core.robot/src/main/java/com/flash3388/flashlib/robot/control/PidController.java

```diff
@@ -15,16 +13,16 @@
  * and a measured process variable and applies a correction based on proportional, integral, and derivative 
  * terms (sometimes denoted P, I, and D respectively) which give their name to the controller type.
  * </p>
- * @author Tom Tzook
- * @since FlashLib 1.0.0
+ * @author Tom Tzook and Daniel Mikhailov
```

Comment:
I've being removing the `author` tags as part of 3.0.0. So remove this.

Reply:
ok